acceptance-tests/features/add_reference_to_case.feature

```diff
@@ -0,0 +1,44 @@
+@add_ref_to_case
+
+Feature: Add Reference to Case
+  As a conveyancer who has many cases to track I need to be able to mark a case
+  with an internal reference so that I can identify exactly which case on my
+  list matches my own internal case list
+
+Scenario Outline: Conveyancer Enters Reference
+
+  - If a case reference is provided then it will be displayed on the case list
+
+  Given I view the case list
+  And I click on the "Create a new case" link
```

Comment:
Should this step and some of the others in this file be I click on the "Create case" button after our discussion yesterday? Or am I getting confused between buttons?